### PR TITLE
reduce the amount of spelling issues / WORDLIST

### PR DESCRIPTION
**Pull request by m7pr**
The follow-up after https://github.com/insightsengineering/coredev-tasks/issues/235

Final `spelling::spell_check_package()` output. Some or inherited by car::Anova parameters and most of them are surnames which I left

```{R}
  WORD            FOUND IN
Anova           try_car_anova.Rd:31
biomarker       control_step.Rd:16,21,26,32,33,34,41
                extract_rsp_biomarkers.Rd:42
                extract_survival_biomarkers.Rd:39
                fit_rsp_step.Rd:20
                fit_survival_step.Rd:23
                g_step.Rd:19,35
                h_response_biomarkers_subgroups.Rd:19,26,74,78
                h_step.Rd:40,72,82,97
                h_survival_biomarkers_subgroups.Rd:19,25,76,79
                h_tab_one_biomarker.Rd:10
                response_biomarkers_subgroups.Rd:33,38
                survival_biomarkers_subgroups.Rd:36,41,51
                NEWS.md:322
Biomarker       h_response_biomarkers_subgroups.Rd:8
                h_survival_biomarkers_subgroups.Rd:8
                h_tab_one_biomarker.Rd:5
                response_biomarkers_subgroups.Rd:6
                survival_biomarkers_subgroups.Rd:6
biomarkers      extract_rsp_biomarkers.Rd:39
                extract_survival_biomarkers.Rd:37
                h_response_biomarkers_subgroups.Rd:51,72
                h_survival_biomarkers_subgroups.Rd:53,73
Biomarkers      extract_rsp_biomarkers.Rd:5
                extract_survival_biomarkers.Rd:5
Bové           tern-package.Rd:24
Clopper         h_proportions.Rd:64
coercible       h_cox_regression.Rd:41,45
Forkers         README.md:75,85
funder          tern-package.Rd:36
Haenszel        h_prop_diff_test.Rd:35
Hauck           h_prop_diff.Rd:59
Hoffmann        tern-package.Rd:36
Jeffreys        h_proportions.Rd:73,74
Kaplan          g_km.Rd:5,109,111,114,116
                h_ggkm.Rd:61,63,71
                h_km_layout.Rd:19,27
                NEWS.md:560
                README.md:30
                tern.Rmd:60
Kenward         try_car_anova.Rd:18
layouting       abnormal.Rd:78
                abnormal_by_baseline.Rd:78
                abnormal_by_marked.Rd:74
                abnormal_by_worst_grade.Rd:69
                abnormal_by_worst_grade_worsen.Rd:75
                analyze_vars_in_cols.Rd:59
                compare_variables.Rd:128
                count_cumulative.Rd:85
                count_missed_doses.Rd:73
                count_occurrences.Rd:101
                count_occurrences_by_grade.Rd:104,110
                count_patients_events_in_cols.Rd:71
                count_patients_with_event.Rd:89
                count_patients_with_flags.Rd:91
                count_values_funs.Rd:96
                cox_regression.Rd:120,147
                estimate_multinomial_rsp.Rd:63
                estimate_proportions.Rd:105
                incidence_rate.Rd:99
                logistic_regression_cols.Rd:15
                odds_ratio.Rd:94
                prop_diff.Rd:98
                prop_diff_test.Rd:91
                split_cols_by_groups.Rd:24
                summarize_ancova.Rd:111
                summarize_change.Rd:62
                summarize_colvars.Rd:32
                summarize_glm_count.Rd:106
                summarize_logistic.Rd:25
                summarize_num_patients.Rd:110,116
                summarize_patients_exposure_in_cols.Rd:108,115
                summarize_variables.Rd:243
                survival_coxph_pairwise.Rd:106
                survival_time.Rd:83
                survival_timepoint.Rd:146
poisson         h_glm_count.Rd:42,71,76
pre             abnormal_by_baseline.Rd:46
                abnormal_by_worst_grade.Rd:79,89
                compare_variables.Rd:177,181
                cox_regression.Rd:104
                h_ggkm.Rd:27
                h_grob_tbl_at_risk.Rd:10
                h_km_layout.Rd:10
                h_xticks.Rd:10
                s_cox_multivariate.Rd:38
                summarize_variables.Rd:294,301
                NEWS.md:405
                tern.Rmd:54,87
                tern_formats.Rmd:198,278
quasipoisson    h_glm_count.Rd:42,73,76
repo            NEWS.md:84
                README.md:9,15,83,85
responder       argument_convention.Rd:76
                check_diff_prop_ci.Rd:10,24
                estimate_proportions.Rd:48
                h_prop_diff.Rd:30
                h_proportions.Rd:33
                h_response_subgroups.Rd:32
                prop_diff.Rd:109
                NEWS.md:436,438,439
                tables.Rmd:358
responders      estimate_proportions.Rd:113
                odds_ratio.Rd:126
                response_subgroups.Rd:28,29
                NEWS.md:201
                tables.Rmd:352,353,367
Sabanés        tern-package.Rd:24
Satterthwaite   try_car_anova.Rd:19
Schouten        h_prop_diff_test.Rd:38,93
                NEWS.md:456,464
                tables.Rmd:427
subtable        score_occurrences.Rd:36,60
subtables       response_biomarkers_subgroups.Rd:48
                score_occurrences.Rd:48,60
                survival_biomarkers_subgroups.Rd:57
TLG             description:1
                NEWS.md:596
                README.md:68
TLGs            title:1
                NEWS.md:533,584
unformatted     cox_regression.Rd:275
unstratified    h_tbl_coxph_pairwise.Rd:37
                odds_ratio.Rd:106
                prop_diff.Rd:118
                s_cox_multivariate.Rd:37
                survival_coxph_pairwise.Rd:114
                NEWS.md:30
```

### CODE REVIEW

**Comment by m7pr** [comment]
cc @edelarua @Melkiades

**Comment by shajoezhu** [comment]
Thanks @m7pr , i m blocking this after the release

**Comment by m7pr** [comment]
@shajoezhu maybe you can take the benefit of this change, before the release? It mostly fixes spelling issues and cleans up the documenation

File: R/summarize_colvars.R [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #' Adding this function to an `rtable` layout will summarize the given variables, arrange the output
 #' in columns, and add it to the table layout.
 #'
-#' @seealso [rtables::split_cols_by_multivar()] and [analyze_colvars_functions].
+#' @seealso [rtables::split_cols_by_multivar()] and [`analyze_colvars_functions`].
```
Comment (Melkiades) [inline R/summarize_colvars.R:22]: Does this lose the link?

File: R/g_lineplot.R [hunk 2]
```diff
@@ -471,7 +471,7 @@ h_format_row <- function(x, format, labels = NULL) {
 #' @param x (`character`)\cr x variable name.
 #' @param y (`character`)\cr y variable name.
 #' @param strata (`character` or `NA`)\cr strata variable name.
-#' @param paramcd (`character` or `NA`)\cr paramcd variable name.
+#' @param paramcd (`character` or `NA`)\cr `paramcd` variable name.
 #' @param y_unit (`character` or `NA`)\cr y_unit variable name.
```
Comment (Melkiades) [inline R/g_lineplot.R:475]: why not `y_unit` too?

File: NEWS.md [hunk 3]
```diff
@@ -12,7 +12,7 @@
 * Fixed bug in `g_lineplot` preventing lines from being added to the plot when midpoint statistic calculations result in `NA` value(s).
```
Comment (Melkiades) [inline NEWS.md:12]: As we are there, could you fix also this grammar action ("being added" may be passive voice)? Thanks ^^

**Review by Melkiades — COMMENTED** [review]
In general, it seems all fine. For me, it is ok to merge it in before release too. Thank you for polishing the `WORDLIST`

File: NEWS.md (hunk 3 above)
Comment (m7pr) [reply NEWS.md:12]: @Melkiades sure :)

**Review by edelarua — APPROVED** [review]
Lgtm! The `tern` submission to CRAN is done so this should be fine to go in now.

**Comment by shajoezhu** [comment]
> Lgtm! The `tern` submission to CRAN is done so this should be fine to go in now.

Yes, definitely! Thanks a lot guys!

File: NEWS.md (hunk 3 above)
Comment (m7pr) [reply NEWS.md]: 565019d916cb720769ebbc3164e731be29eabb9e

File: R/g_lineplot.R (hunk 2 above)
Comment (m7pr) [reply R/g_lineplot.R]: good catch - aed901b0ca9f1a19567c814b1696b6e2796a86b4

**Pull request merged**